lib/active_model/serializer/adapter/attributes.rb

```diff
@@ -2,47 +2,54 @@ module ActiveModel
   class Serializer
     module Adapter
       class Attributes < Base
+        attr_accessor :_include_tree
+
         def initialize(serializer, options = {})
           super
-          @include_tree = IncludeTree.from_include_args(options[:include] || '*')
+          self._include_tree = IncludeTree.from_include_args(options[:include] || '*')
```

Comment:
Nope, it has not been settled yet, and may never be. Plus if you change it here, you'd have to change it in JsonApi as well, and it's outside the scope of this PR.